### ISSUE TITLE
IPTT: Include change log as final tab in workbook export

### ISSUE THREAD

**Issue by jennyhavah** [mercycorps/TolaActivity#1319]
**Depends on:** Partner Access
**Follow-up task for:** #346

# Issue
With Partner Access, we are implementing a change log that is exportable to Excel. USAID and other donors require the change log to be included in the IPTT workbook. So I'd like to include our change log as part of our "All program data" export as the final tab. 

# Acceptance criteria

- [x] 1. When I export "All program data", the final tab of the workbook is called "Change log"
- [x] 2. The change log tab includes all content I see in the web version.

**Comment by jennyhavah** [comment]
@camgnostic Please make "Change Log" sentence case.

![image](https://user-images.githubusercontent.com/33670923/58988216-00379e80-8796-11e9-8c99-f6d4429d0ba9.png)

**Comment by jennyhavah** [comment]
Passed! Sheet name is now "Change log", sentence case.

![image](https://user-images.githubusercontent.com/33670923/59119703-47de3780-8908-11e9-96b1-ad71cdd0c7a0.png)
